doc/geany.txt

```diff
@@ -538,6 +538,16 @@ there is a selection, only the selected text is copied. This can be
 useful when making temporary copies of text or for creating
 documents with similar or identical contents.
 
+Automatic filename insertion on `Save As...`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If a document is saved via `Document->Save As...` then the filename can
+automatically be inserted into the comment header. This only happens if
```

Comment:
replace ". This only happens if geany finds a" with "replacing" so you get "...comment header replacing text like..."

Also just a note "Geany" is always capitalised